qa/TL1_paddle_tsm/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
```

Comment:
It looks like this particular test fails. Can you check it?

Reply:
@JanuszL
Can you  provide some more details? I rebased the commits to latest master and rerun the test a couple of times, and they all exit with no error. 

On my setup the prediction is pretty stable ("paragliding" ), but there more be some randomness as it is not from Kinectic dataset, so I remove the checking anyway. 

Could you please restart the test and see if it still fails?  And if it does, maybe provide some error logs?

Reply:
Apparently this test was giving the label `sticking_tongue_out` which seems reasonable looking at the video sample. I added this case as a correct output